Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM hexpm/elixir:1.14.0-erlang-25.0.4-debian-bullseye-20220801
```

Comment:
There doesn't seem to be a debian based docker image for 1.15/26, but I see ubuntu has been used in prior version updates here

Reply:
Ah I was looking at plain 26.0, I'll update to 26.0.1 and use the debian images

Reply:
debian buster is older than debian bullseye.  If you could use [1.15.0-erlang-26.0.1-debian-bookworm-20230612](https://hub.docker.com/layers/hexpm/elixir/1.15.0-erlang-26.0.1-debian-bookworm-20230612/images/sha256-025bc49b88562025fac1479e338592e469670076bcbc43542d5e8e9ca144f31d?context=explore) that would be a preferable option.

Reply:
Good idea, I will update the PRs